dislib/data/array.py

```diff
@@ -8,6 +8,7 @@
 from pycompss.api.parameter import Type, COLLECTION_IN, Depth, \
     COLLECTION_OUT, INOUT
 from pycompss.api.task import task
+import scipy
```

Comment:
You could replace this import with `from scipy.special import gammaln` as in the test file.